test/support/factory.ex

```diff
@@ -577,7 +578,7 @@ defmodule Trento.Factory do
 
   def tag_factory do
     %Tag{
-      value: sequence(:value, &"#{Faker.Beer.hop()}0#{&1}"),
+      value: sequence(:value, &"#{Faker.Color.name()}0#{&1}"),
```

Comment:
As a home brewer (and starwars fan) i'm very offended by this change (it's fine, just kidding :rofl:)

Reply:
Beers fans aside XD
This was needed, as some beer names have spaces, and it was breaking validation rules